packages/ui/tsconfig.json

```diff
@@ -1,14 +1,9 @@
 {
-  "ts-node": {
-    "compilerOptions": {
-      "module": "CommonJS"
-    }
-  },
   "compilerOptions": {
     "target": "es5",
     "module": "esnext",
     "strict": false,
-    "jsx": "preserve",
```

Comment:
curious what this did before?

Reply:
This is not a correct config. It tells typescript to preserve the JSX syntax. We do not use any JSX in ui package, so this setting is useless but would like to make it consistent now

Reply:
Thanks for doing that, was going to make the same change in the dev command PR